docs/pages/build-reference/caching.mdx

```diff
@@ -85,4 +85,24 @@ To using our CocoaPods cache server for your builds set the `EAS_BUILD_DISABLE_C
   }
   /* @hide ... */ /* @end */
 }
+```
+
+EAS Build allows you to cache the **Podfile.lock** file to provide consistent results across builds when using the managed workflow.
```

Comment:
```suggestion
EAS Build allows you to cache the **Podfile.lock** file to provide consistent results across builds when using the managed workflow.
```

Do we need to specify managed workflow here? I think it might be confusing since we generally don't specify managed/bare workflow in EAS docs. I understand (not sure) that the behavior can be different for bare workflows. Although, I'd be interested in learning what are the differences between managed and bare workflows.

 Can we use more generic term like "your project"?